pieces/regex_json/python/main.py

```diff
@@ -31,11 +36,15 @@ def calculate_range(revenue: str) -> float:
     return annual_revenue_lower, annual_revenue_upper
 
 
-def run() -> None:
+def run() -> list[dict[str, Any]]:
     data = get_data()
-    assert data
+    if not data:
+        raise ValueError("No data found")
```

Comment:
I switch this here because `assert` can be bypassed. `python -O main.py` for example would bypass the assert.